lib/modules/manager/gitlabci/utils.spec.ts

```diff
@@ -50,6 +50,8 @@ describe('modules/manager/gitlabci/utils', () => {
       ${'multiple aliases'}        | ${{ foo: 'foo.registry.com', bar: 'bar.registry.com' }} | ${'foo/image:1.0'}            | ${{ depName: 'foo/image', packageName: 'foo.registry.com/image', currentValue: '1.0', autoReplaceStringTemplate: `foo/image${versionAndDigestTemplate}` }}
       ${'aliased variable'}        | ${{ $CI_REGISTRY: 'registry.com' }}                     | ${'$CI_REGISTRY/image:1.0'}   | ${{ depName: '$CI_REGISTRY/image', packageName: 'registry.com/image', currentValue: '1.0', autoReplaceStringTemplate: `$CI_REGISTRY/image${versionAndDigestTemplate}` }}
       ${'variables with brackets'} | ${{ '${CI_REGISTRY}': 'registry.com' }}                 | ${'${CI_REGISTRY}/image:1.0'} | ${{ depName: '${CI_REGISTRY}/image', packageName: 'registry.com/image', currentValue: '1.0', autoReplaceStringTemplate: `$\{CI_REGISTRY}/image${versionAndDigestTemplate}` }}
+      ${'variables with default and slash'} | ${{ '${CI_REGISTRY:-}': 'registry.com' }}      | ${'${CI_REGISTRY:-}/image:1.0'} | ${{ currentValue: '1.0' }}
```

Comment:
Expand this test to include `depName`, `packageName` and `autoReplaceStringTemplate`.

I think the expected `depName` will be: `${CI_REGISTRY:-}/image`.
But when running the tests I get: `"depName": "${CI_REGISTRY",`

Reply:
please fill expected `depName`, `packageName` and `autoReplaceStringTemplate`